sparse_strips/vello_cpu/tests/mod.rs

```diff
@@ -0,0 +1,7 @@
+// Copyright 2025 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![allow(missing_docs, reason = "we don't need docs for testing")]
+#![allow(clippy::cast_possible_truncation, reason = "not critical for testing")]
+
```

Comment:
We should probably have a tiny bit of docs here. E.g. mentioning that `util` is for shared things.

(There are arguments for splitting `util` into e.g. `harness` and something else, say `shared`, `content`, or indeed `util`)